de-lib/src/main/java/org/iplantc/de/notifications/client/utils/NotifyInfo.java

```diff
@@ -53,8 +53,10 @@ protected void positionAnnouncer() {
 
         IplantAnnouncement popup = announcements.peek();
 
+        //we support a minimum of 800 x 600
```

Comment:
Ohh ya good catch. I should test with 1024 x 768 just to make the notification is still visible.